pom.xml

```diff
@@ -145,6 +145,11 @@
             <artifactId>validation-api</artifactId>
         </dependency>
         <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>annotations</artifactId>
+            <version>3.0.0</version>
```

Comment:
@cvrebert check this: http://stackoverflow.com/questions/30702059/how-to-ignore-or-fix-the-duplicate-classes-warning-in-qulice/30814032#30814032 you can completely suppress findbugs, just by using `<exclusion>findbugs:.*</exclusion>`, but I don't think it's a good idea.. maybe let's do what findbugs is asking us to do?